src/config.rs

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: BSD-2-Clause
+
+use std::path::PathBuf;
+
+use pyo3::prelude::*;
+
+#[pyclass(eq, eq_int)]
+#[derive(PartialEq, Clone)]
+pub enum Backend {
+    WINCH,
+    CRANELIFT,
+}
+
+impl From<::wasmtime::Strategy> for Backend {
+    fn from(value: ::wasmtime::Strategy) -> Self {
+        match value {
+            ::wasmtime::Strategy::Cranelift => Backend::CRANELIFT,
+            _ => Backend::WINCH,
+        }
+    }
+}
+
+#[pyclass(eq, eq_int)]
+#[derive(PartialEq, Clone)]
+pub enum OptLevel {
+    NONE,
+    SPEED,
```

Comment:
I feel like there should also be `SpeedAndSize` for sake of completeness https://docs.wasmtime.dev/api/cranelift_codegen/settings/enum.OptLevel.html